dex/testing/eth/create-node.sh

```diff
@@ -56,19 +56,17 @@ function mine() {
 }
 EOF
 
-# Write password file to unlock accounts later.
-cat > "${GROUP_DIR}/password" <<EOF
+  # Write password file to unlock accounts later.
+  cat > "${GROUP_DIR}/password" <<EOF
 $CHAIN_PASSWORD
-$ADDRESS_PASSWORD
```

Comment:
It does not appear that this was unlocking the account. We will pass the password when sending funds.